### ISSUE TITLE
Cannot scroll to the last item, when variableWidth: true

### ISSUE THREAD

**Issue by inliner** [kenwheeler/slick#2501]
I think I have similar issue as #2491, but it's not the same, so I decided to open a new one.
If I set `variableWidth: true` and the number of slidesToShow is greater than can fit on screen, I cannot scroll to the last item(s).

====================================================================

use this jsFiddle to reproduce your bug: 
http://jsfiddle.net/q6apj70t/1/

====================================================================

#### Steps to reproduce the problem

1. Create a slider with variableWidth: true, infinite: false, slidesToShow is greater than can fit on screen (e.g 10)
2. Click the "Next" button all the way to the end when it gets disabled.
3. See that the last item is not visible

====================================================================

#### What is the expected behaviour?

You must be able to scroll to the last item.

====================================================================

#### What is observed behaviour?

You cannot scroll to the last item. 

====================================================================

#### More Details

- Which browsers/versions does it happen on? I tested in Chrome, FF, Edge, iOS Safari
- Which jQuery/Slick version are you using? jQuery 3.1 / Slick 1.6
- Did this work before? No

**Comment by Meh0** [comment]
Set slides as this :

```
slidesToShow: 1,
slidesToScroll: 1
```

**Comment by inliner** [comment]
No, this won't help. If I set `slidesToShow: 1` and scroll to the very last item it allows me to continue scrolling till there's only 1 item visible in container while it can actually hold 5, 7, whatever.

**Comment by Meh0** [comment]
Oh I see, then you would like the last item to stay stuck to the right border ?

I guess it's a collateral of the variableWidth. 

You could write a few lines of JS to correct that, but I don't know if we can consider this as an issue, as not everyone would need this behaviour. I'm not the developper of this library though.
